requirements/common_constraints.txt

```diff
@@ -25,4 +25,4 @@ elasticsearch<7.14.0
 
 # Cause: https://github.com/openedx/edx-lint/issues/458
 # This can be unpinned once https://github.com/openedx/edx-lint/issues/459 has been resolved.
-pip<24.3
```

Comment:
Wouldn’t you want to remove the comment? Ultimately, if this lands, we would close the ticket as well.

Reply:
The common constraints file removal is via `sed` in the Makefile(how we've done this in the past), the comment will get removed when we actually drop this upstream in edx-lint.